tests/envs/mujoco/test_mujoco_v5.py

```diff
@@ -0,0 +1,676 @@
+            -info["reward_ctrl"] + -info["reward_contact"]
+        )
+
+    env = gym.make(f"HalfCheetah-{version}")
+    env.reset()
+    for _ in range(NUM_STEPS):
+        _, reward, _, _, info = env.step(env.action_space.sample())
+        assert reward == info["reward_forward"] + info["reward_ctrl"]
+
+    env = gym.make(f"Hopper-{version}")
+    env.reset()
+    for _ in range(NUM_STEPS):
+        _, reward, _, _, info = env.step(env.action_space.sample())
+        assert (
+            reward
+            == info["reward_forward"] + info["reward_survive"] + info["reward_ctrl"]
+        )
+
+    env = gym.make(f"Humanoid-{version}")
+    env.reset()
+    for _ in range(NUM_STEPS):
+        _, reward, _, _, info = env.step(env.action_space.sample())
+        assert reward == (info["reward_forward"] + info["reward_survive"]) + (
+            info["reward_ctrl"] + info["reward_contact"]
+        )
+
+    env = gym.make(f"HumanoidStandup-{version}")
+    env.reset()
+    for _ in range(NUM_STEPS):
+        _, reward, _, _, info = env.step(env.action_space.sample())
+        assert (
+            reward
+            == info["reward_linup"]
+            + info["reward_quadctrl"]
+            + info["reward_impact"]
+            + 1
+        )
+
+    env = gym.make(f"InvertedDoublePendulum-{version}")
+    env.reset()
+    for _ in range(NUM_STEPS):
+        _, reward, _, _, info = env.step(env.action_space.sample())
+        assert (
+            reward
+            == info["reward_survive"]
+            + info["distance_penalty"]
+            + info["velocity_penalty"]
+        )
+
+    env = gym.make(f"InvertedPendulum-{version}")
+    env.reset()
+    for _ in range(NUM_STEPS):
+        _, reward, _, _, info = env.step(env.action_space.sample())
+        assert reward == info["reward_survive"]
+
+    env = gym.make(f"Pusher-{version}")
+    env.reset()
+    for _ in range(NUM_STEPS):
+        _, reward, _, _, info = env.step(env.action_space.sample())
+        assert reward == info["reward_dist"] + info["reward_ctrl"] + info["reward_near"]
+
+    env = gym.make(f"Reacher-{version}")
+    env.reset()
+    for _ in range(NUM_STEPS):
+        _, reward, _, _, info = env.step(env.action_space.sample())
+        assert reward == info["reward_dist"] + info["reward_ctrl"]
+
+    env = gym.make(f"Swimmer-{version}")
+    env.reset()
+    for _ in range(NUM_STEPS):
+        _, reward, _, _, info = env.step(env.action_space.sample())
+        assert reward == info["reward_forward"] + info["reward_ctrl"]
+
+    env = gym.make(f"Walker2d-{version}")
+    env.reset()
+    for _ in range(NUM_STEPS):
+        _, reward, _, _, info = env.step(env.action_space.sample())
+        assert (
+            reward
+            == info["reward_forward"] + info["reward_survive"] + info["reward_ctrl"]
+        )
+
+
+# Note: the environtments "HalfCheetah", "Pusher", "Swimmer", are identical between `v4` & `v5` (excluding `info`)
+def test_identical_behaviour_v45():
```

Comment:
Is it possible to make this a pytest.mark.parameterize for the env name then a dictionary of arguments for the `check_environment_match` function?